keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/verification/VerificationRepository.kt

```diff
@@ -0,0 +1,33 @@
+package com.netflix.spinnaker.keel.api.verification
+
+import com.netflix.spinnaker.keel.api.Environment
+import com.netflix.spinnaker.keel.api.Verification
+import com.netflix.spinnaker.keel.api.artifacts.DeliveryArtifact
+import java.time.Instant
+
+interface VerificationRepository {
+  fun getState(
```

Comment:
Note that a verification may have run multiple times (e.g., test suite failed, user triggered a re-run, which passed). This means that there could be multiple VerificationState objects associated with a (verification, environment, artifact, version) tuple.

Reply:
Not trying to solve for that yet